mitxgraders/helpers/get_number_of_args.py

```diff
@@ -161,9 +161,6 @@ def get_number_of_args(callable_obj):
     raise an error; in Python 3, the number of arguments of __init__ is returned.
 
     Works on built-in functions (assuming their docstring is correct)
-    >>> import math
```

Comment:
Apparently my version of python 3 doesn't like this...

Reply:
Can you elaborate? What's the error? What version of Python 3 are you in?

Reply:
Python 3.6.4.

[doctest] mitxgraders.helpers.get_number_of_args.get_number_of_args:

```
163     Works on built-in functions (assuming their docstring is correct)
164     >>> import math
165     >>> get_number_of_args(math.sin)
UNEXPECTED EXCEPTION: ValueError('no signature found for builtin <built-in function sin>',)
Traceback (most recent call last):

  File "/Users/jolyon/anaconda/lib/python3.6/doctest.py", line 1330, in __run
    compileflags, 1), test.globs)

  File "<doctest mitxgraders.helpers.get_number_of_args.get_number_of_args[12]>", line 1, in <module>

  File "/Users/jolyon/gitrepos/mitx-grading-library-py3/mitxgraders/helpers/get_number_of_args.py", line 179, in get_number_of_args
    return get_number_of_args_py3(callable_obj)

  File "/Users/jolyon/gitrepos/mitx-grading-library-py3/mitxgraders/helpers/get_number_of_args.py", line 119, in get_number_of_args_py3
    params = inspect.signature(callable_obj).parameters

  File "/Users/jolyon/anaconda/lib/python3.6/inspect.py", line 3036, in signature
    return Signature.from_callable(obj, follow_wrapped=follow_wrapped)

  File "/Users/jolyon/anaconda/lib/python3.6/inspect.py", line 2786, in from_callable
    follow_wrapper_chains=follow_wrapped)

  File "/Users/jolyon/anaconda/lib/python3.6/inspect.py", line 2265, in _signature_from_callable
    skip_bound_arg=skip_bound_arg)

  File "/Users/jolyon/anaconda/lib/python3.6/inspect.py", line 2090, in _signature_from_builtin
    raise ValueError("no signature found for builtin {!r}".format(func))

ValueError: no signature found for builtin <built-in function sin>
```